jobs/kola-openstack.Jenkinsfile

```diff
@@ -92,20 +92,22 @@ lock(resource: "kola-openstack-${params.ARCH}") {
         withCredentials([file(variable: 'OPENSTACK_KOLA_TESTS_CONFIG',
                               credentialsId: 'openstack-kola-tests-config')]) {
 
-            stage('Upload/Create Image') {
-                // Create the image in OpenStack
-                shwrap("""
-                # First delete it if it currently exists, then create it.
-                ore openstack --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG} \
-                    --region=${region} \
-                    delete-image --id=${openstack_image_name} || true
-                ore openstack --config-file=\${OPENSTACK_KOLA_TESTS_CONFIG} \
-                    --region=${region} \
-                    create-image --file=${openstack_image_filepath} \
-                    --name=${openstack_image_name} --arch=${params.ARCH}
-                """)
+            lock(resource: "kola-cloud-buildfetch") {
```

Comment:
This lock needs to go higher up around the `stage('Fetch Metadata/Image')` stage and not here.

Reply:
Ouch, that sheds some light on my attention to detail this evening. My bad, thanks for catching this.

Reply:
Resolved in: https://github.com/coreos/fedora-coreos-pipeline/commit/f2cf30f234696e6a0e84ce5e5a7ff4d6d4b8f2bd